packages/@aws-cdk/aws-ec2/lib/security-group-lookup.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Properties for looking up an existing SecurityGroup.
+ *
+ * The combination of properties must specify filter down to exactly one
+ * security group, otherwise an error is raised.
+ */
+export interface SecurityGroupLookupOptions {
+  /**
+   * The VPC Id the security group belongs to
+   *
+   */
+  readonly vpcId: string;
```

Comment:
What if the `vpcId` needs to be looked up as well? Wouldn't an `IVpc` make more sense?